homeassistant/components/devolo_home_control/strings.json

```
@@ -5,18 +5,18 @@
       "already_configured": "This Home Control Central Unit is already in use."
```

Please replace this key as well, thanks :+1: 
```suggestion
      "already_configured": "[%key:common::config_flow::abort::already_configured_device%]"
```

---

I changed it to already_configured_account, as we configure one gateway that then brings multiple devices.